.editorconfig

```diff
@@ -11,7 +11,7 @@ end_of_line = lf
 [Makefile]
 indent_style = tab
 
-[*.{xml,js,json,yaml}]
```

Comment:
@nhsd-jack-wainwright Is this ok? It seems to make editing the xml files easier in my IntelliJ. Without this change, I have to add extra indents when pasting xml snippets copied from elsewhere in the same file.

Reply:
I haven't noticed this but I don't use intellij as part of updating these files. Does this impact the formatters that apply on commit?

Reply:
I don't think so. The files in this change will have been run through the formatters.

Reply:
@nhsd-david-wass do you use IntelliJ? Does this change look ok to you? We seem to use 4 spaces for an indent in xml.

Reply:
Yes - I use intellij - and spend ages reformatting xml files when I copy and paste into them.
If this change improves that all the better

Reply:
It seems to for me. Ok, we'll go with it.